Update multiple consumers info (#9259)
magento/devdocs: src/guides/v2.4/config-guide/mq/manage-message-queues.md

*  `consumers` - An array of strings specifying which consumer(s) to run. An empty array runs *all* consumers.
*  `multiple_processes` - An array of key-value pairs specifying which consumer to run in how many processes.

   {:.bs-callout-info}

>    We do not recommned to run multiple consumers on Mysql operated queue. See [`how to change queue from Mysql to Rabbit`]({{ page.baseurl }}/guides/v2.3/extension-dev-guide/message-queues/message-queues.html#change-message-queue-from-mysql-to-amqp)

Reviewer: ```suggestion
   We do not recommend running multiple consumers on a Mysql operated queue. See [`how to change queue from Mysql to Rabbit`]({{ page.baseurl }}/guides/v2.3/extension-dev-guide/message-queues/message-queues.html#change-message-queue-from-mysql-to-amqp)
```
